jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -634,15 +643,38 @@ def serialize_script(script):
     else:
         return result
 
-
 def mk_multisig_script(pubs, k):
     """ Given a list of pubkeys and an integer k,
     construct a multisig script for k of N, where N is
     the length of the list `pubs`; script is returned
     as hex string.
     """
-    return serialize_script([k] + pubs + [len(pubs)]) + 'ae'
+    return serialize_script([k] + pubs + [len(pubs)]) \
+        + 'ae' #OP_CHECKMULTISIG
```

Comment:
I guess for consistency use `btscript` here also, but a higher level comment:
since you are not adding multisig in this PR and we are not yet using it anywhere, I have deliberately ditched it from `jmbitcoin` in the #536 backend update. the functionality is all in the backend of course, but i won't re add msig script creation functions until we actually have a use for them, else it's just more code to maintain for no purpose.

Reply:
I'm going to skip doing this because we'll remove it.